operations/mimir/alertmanager.libsonnet

```diff
@@ -11,7 +11,7 @@
 
   alertmanager_args::
     $._config.grpcConfig +
-    $._config.alertmanagerStorageClientConfig +
```

Comment:
I think you need to merge `$.storageConfig` too, before `$._config.alertmanagerStorageConfig` (so that `alertmanagerStorageConfig` can override it).

Reply:
Right, thank you. Fixed in https://github.com/grafana/mimir/pull/3257/commits/00d0f3f976850fb51de72eb1d0f45ba0da0b9d29